teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li><s>Often</s>, <strong>Docking</strong> programs <strong>can consider some residue</strong> sidechains flexible during docking calculations to account for binding&nbsp;pocket flexibility.</li></ul><p> </p><p><em>Some bullet points are misplaced. This might be due reviewnb again, but double check!</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>